boards/risc-v/nr5m100/nr5m100-nexys4/README.txt

```diff
@@ -143,7 +143,7 @@ Debugger
      shadowed variable warnings, so disable the -Werror flag also:
 
      cd openocd
-     sed -i 's/ -Werror//g' configure.ac
+     sed -i.bak 's/ -Werror//g' configure.ac
```

Comment:
don't need to moidfy README.txt?

Reply:
why not?

Reply:
I think it's a demo, but anyway it's fine to modify it.